faq/index.adoc

```diff
@@ -0,0 +1,92 @@
+= Adoptium Frequently Asked Questions
+
+We have completed our first releases at Eclipse Temurin and will be updating this
+FAQ as and when things are completed and available. If you want to talk to us or ask
+additional questions the best place is our
+https://adoptium.net/slack.html[slack instance]
```

Comment:
we have not defined the best practices for formatting documentation. In general I prefer 1 sentence per line. What kind of format have you used? Why do you always put links in newlines?

Reply:
I was wrong: reformatted to one sentence per line